bin/pakiti-client

```diff
@@ -238,8 +237,38 @@ sub find_packages ($) {
     my($data) = @_;
     my($cmd, $format, @list);
     my(@output);
-
-    # Red Hat packages
+	
+	#We check if we got called with svmon parameter
+	$cmd = $Option{"svmonpath"};	
+	#If we dont have the svmon execution parameter we use the default one
+	if(!$cmd) {
+		$cmd = 'svmon'
+	}
```

Comment:
Please remove the default setting of svmon since that would break systems without the command. What you want to do is to rename the option to just svmon (i.e. to be the same as the command).